react/week2/my-app/src/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import Counter from "./count";
+import TodoList from'./render';
+
+const root = document.getElementById("root");
+
+const  todoList = [
+  {
+    id: 1,
+    description: "Get out of bed"
+  },
+  {
+    id: 2,
+    description: "Brush teeth"
+  },
+  {
+    id: 3,
+    description: "Eat breakfast"
+  }
+]
+
+class RenderTodo extends React.Component {
+  state = {isShow: true}
+ 
+  render() {
+    const { isShow } = this.state;
+    
+
+    return (
+      <div>
+        {isShow ? <Counter /> : <p>nothing to show</p>}
+      </div>
+   ); 
+ 
+ 
+  } 
+}
```

Comment:
I doesn't look like you allow the user to interact with the state of this component. I would consider if there is really any value in this component and potentially remove it to simply your code.